storybook/package.json

```diff
@@ -26,34 +26,32 @@
     "@amsterdam/design-system-tokens": "workspace:*",
     "@babel/core": "7.24.0",
     "@mdx-js/react": "2.3.0",
-    "@storybook/addon-a11y": "7.6.17",
-    "@storybook/addon-actions": "7.6.17",
-    "@storybook/addon-docs": "7.6.17",
-    "@storybook/addon-essentials": "7.6.17",
-    "@storybook/addon-interactions": "7.6.17",
-    "@storybook/addon-links": "7.6.17",
-    "@storybook/addon-themes": "7.6.17",
-    "@storybook/addons": "7.6.17",
-    "@storybook/api": "7.6.17",
-    "@storybook/blocks": "7.6.17",
-    "@storybook/components": "7.6.17",
-    "@storybook/core-events": "7.6.17",
-    "@storybook/manager-api": "7.6.17",
+    "@storybook/addon-a11y": "8.0.0",
+    "@storybook/addon-actions": "8.0.0",
+    "@storybook/addon-docs": "8.0.0",
+    "@storybook/addon-essentials": "8.0.0",
+    "@storybook/addon-interactions": "8.0.0",
+    "@storybook/addon-links": "8.0.0",
+    "@storybook/addon-themes": "8.0.0",
+    "@storybook/blocks": "8.0.0",
+    "@storybook/components": "8.0.0",
+    "@storybook/core-events": "8.0.0",
+    "@storybook/manager-api": "8.0.0",
     "@storybook/mdx2-csf": "1.1.0",
-    "@storybook/preview-api": "7.6.17",
-    "@storybook/react": "7.6.17",
-    "@storybook/react-vite": "7.6.17",
-    "@storybook/testing-library": "0.2.2",
-    "@storybook/theming": "7.6.17",
+    "@storybook/preview-api": "8.0.0",
+    "@storybook/react": "8.0.0",
+    "@storybook/react-vite": "8.0.0",
+    "@storybook/theming": "8.0.0",
     "@types/react": "18.2.61",
     "@vitejs/plugin-react": "4.2.1",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "require-from-string": "2.0.2",
-    "storybook": "7.6.17",
+    "storybook": "8.0.0",
     "vite": "5.1.4"
   },
   "dependencies": {
-    "remark-gfm": "3.0.1"
+    "@storybook/test": "8.0.0",
```

Comment:
We don't seem to be using this (yet)

Reply:
Sure, I've added to replace @storybook/testing-library, which was also unused I guess? After removing my interaction tests for Dialog?

Reply:
Yeah, I think so